src/types.rs

```diff
@@ -1080,6 +1164,11 @@ impl TypesBuilder {
             LLVMTypeKind::LLVMMetadataTypeKind => self.metadata_type(),
             LLVMTypeKind::LLVMLabelTypeKind => self.label_type(),
             LLVMTypeKind::LLVMTokenTypeKind => self.token_type(),
+            #[cfg(feature = "llvm-16-or-greater")]
+            LLVMTypeKind::LLVMTargetExtTypeKind => {
+                todo!()
```

Comment:
From a quick look, there don't appear to be LLVM-C APIs for accessing this type's members, yet.